app/client/lib/highlight.js

```diff
@@ -1,5 +1,4 @@
 var find = require('findandreplacedomtext')
-var css = require('dom-css')
 
 // Highlights all the words in text inside a node
 function Highlight (node, text, wrapper) {
```

Comment:
Good point! My version would throw an error if the input was an array.

Looks to me like your suggested fix is perfect 👍 !